Makefile

```diff
@@ -332,6 +334,7 @@ generate-e2e-templates-v1.7: $(KUSTOMIZE)
 .PHONY: lint
 lint: $(GOLANGCI_LINT) ## Lint the codebase
 	$(GOLANGCI_LINT) run -v $(GOLANGCI_LINT_EXTRA_ARGS)
```

Comment:
I think we have to modify pr-golangci-lint.yaml to also run golangci-lint in the test package (compare with the core CAPI action)